### PR TITLE
Update JS collector config --> OTLP exporter config

### PR DESCRIPTION
**Pull request by cartermp**
The way you do it isn't really specific to collectors, so this can be generally useful.

---

Preview: https://deploy-preview-988--opentelemetry.netlify.app/docs/instrumentation/js/exporters/

### CODE REVIEW

File: content/en/docs/instrumentation/js/exporters.md
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:17]: Let's avoid using "e.g." in the docs (yes I know there are other occurrences, but let's avoid introducing new ones).

Maybe:

> To send trace data ... package, such as `@opentelemetry/exporter-trace-otlp-http`.

File: content/en/docs/instrumentation/js/exporters.md
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:20]: I've started an effort to standardize all of these to use the `console` block-code language specifier, and start the command with the `$` prompt.

This is an optional change, I'll be sweeping the docs to change it elsewhere, so it's up to you if you want to do it now.

File: content/en/docs/instrumentation/js/exporters.md
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:23]: And --> Also

File: content/en/docs/instrumentation/js/exporters.md
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:37]: How about just:

> Use environment variables to set ...

**Review by chalin — COMMENTED** [review]
A few minor comments, but otherwise LGTM

**Comment by chalin** [comment]
CC @open-telemetry/javascript-approvers

**Review by chalin — APPROVED** [review]
LGTM

**Comment by chalin** [comment]
@austinlparker - I'll let you have a final look and merge. Thx

**Review by austinlparker — APPROVED** [review]

**Pull request merged**
